rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -136,15 +139,21 @@ If the resulting list of transactions is empty and the `LC` value in the message
 
 #### 5.2.3 Transaction List
 
-When a node receives a `TransactionListQuery` message, it MUST respond with a `TransactionList` message.
+When a node receives a `TransactionListQuery` message, it SHOULD respond with a `TransactionList` message.
 This is a response type message so it MUST include the sent `conversationID`.
 Transactions that resulted from a `TransactionListQuery` MUST have been present in that message.
 Unknown transactions references SHOULD be ignored.
 Transactions that resulted from a `TransactionRangeQuery` MUST have an LC value that is within the requested range.
 If any of these requirements are not met, the entire message MUST be ignored.
+If the resulting list of transactions is empty, a node MAY ignore the message and not send a `TransactionList` message.
+If the list is not empty, it MUST send a response.
+All transactions in the `TransactionList` message MUST be sorted by LC value (lowest first).
+All transactions without a `pal` header MUST be added with their payload.
+If a payload is missing for a transaction, the sender of the `TransactionList` message MUST not add that transaction and any other requested transaction with an LC value equals or higher that that transaction.
```

Comment:
but the other side will keep asking for those TXs. So this is not really a solution?

Reply:
removed